### PR TITLE
Added time metrics to `weather-mv` ee

### PR DESCRIPTION
**Pull request by aniketinfocusp**
Added 
`job_start_time`: timestamp of the beginning of convert to asset
`injestion_time`: timestamp of the start of ingestion into earth engine.

The time stamps are ISO date time strings adjusted to the UCT time zone.

fixes: #360

### CODE REVIEW

File: weather_mv/loader_pipeline/ee.py [hunk 1]
```diff
@@ -602,6 +607,8 @@ def start_ingestion(self, asset_request: t.Dict) -> str:
         """Creates COG-backed asset in earth engine. Returns the asset id."""
         self.check_setup()
 
+        asset_request['properties']['ingestion_time'] = get_utc_timestamp()
```
Comment (deepgabani8) [inline weather_mv/loader_pipeline/ee.py:610]: This is at a good place :)

File: weather_mv/loader_pipeline/util.py [hunk 2]
```diff
@@ -134,6 +134,12 @@ def _check_for_coords_vars(ds_data_var: str, target_var: str) -> bool:
     specified by the user."""
     return ds_data_var.endswith('_'+target_var) or ds_data_var.startswith(target_var+'_')
 
+def get_utc_timestamp() -> str:
+    """Returns the current UTC Timestamp as ISO datetime string."""
+    dt = datetime.datetime.now(timezone.utc)
+    utc_time = dt.replace(tzinfo=timezone.utc)
+
+    return utc_time.isoformat()
```
[the remark this answers is not in the document]
Comment (aniketinfocusp) [reply weather_mv/loader_pipeline/util.py]: updated in  f9003d0

File: weather_mv/loader_pipeline/ee.py [hunk 3]
```diff
@@ -498,6 +500,9 @@ def convert_to_asset(self, queue: Queue, uri: str):
                     with FileSystems().create(target_path) as dst:
                         shutil.copyfileobj(tmp_df, dst, WRITE_CHUNK_SIZE)
 
+            # Adding job_start_time to properites.
+            attrs["job_start_time"] = job_start_time
```
[the remark this answers is not in the document]
Comment (aniketinfocusp) [reply weather_mv/loader_pipeline/ee.py]: updated in f9003d0

File: weather_mv/loader_pipeline/util.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (deepgabani8) [reply weather_mv/loader_pipeline/util.py]: Can use one-liner `datetime.datetime.now().timestamp()`. Does it not work?

File: weather_mv/loader_pipeline/ee.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (deepgabani8) [reply weather_mv/loader_pipeline/ee.py]: :+1:

File: weather_mv/loader_pipeline/util.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (aniketinfocusp) [reply weather_mv/loader_pipeline/util.py]: added in [c0fbe3f](https://github.com/google/weather-tools/pull/361/commits/c0fbe3fe06eebc983cb2dc0741e9555983c0e47e)

**Review by deepgabani8 — APPROVED** [review]

**Pull request merged**